packages/svelte/src/compiler/utils/mapped_code.js

```diff
@@ -398,10 +398,13 @@ export function merge_with_preprocessor_map(result, options, source_name) {
 		// map may contain a different file name. Patch our map beforehand to align sources so merging
 		// with the preprocessor map works correctly.
 		result.map.sources = [file_basename];
-		result.map = apply_preprocessor_sourcemap(
-			file_basename,
+		Object.assign(
```

Comment:
why the change here?

Reply:
`result.map` is now readonly (it is generated lazily in a getter). _technically_ a breaking change but vanishingly unlikely to affect anyone in practice